tests/test_geoprocessing.py

```diff
@@ -248,26 +249,37 @@ def test_reclassify_raster_empty_value_map(self):
     def test_reproject_vector(self):
         """PGP.geoprocessing: test reproject vector."""
         import pygeoprocessing
-        import pygeoprocessing.testing
-        from pygeoprocessing.testing import sampledata
-        reference = sampledata.SRS_WILLAMETTE
-        pixel_size = 30.0
-        n_pixels = 9
-        polygon_a = shapely.geometry.Polygon([
-            (reference.origin[0], reference.origin[1]),
-            (reference.origin[0], -pixel_size * n_pixels+reference.origin[1]),
-            (reference.origin[0]+pixel_size * n_pixels,
-             -pixel_size * n_pixels+reference.origin[1]),
-            (reference.origin[0]+pixel_size * n_pixels, reference.origin[1]),
-            (reference.origin[0], reference.origin[1])])
+
+        # Create polygon shapefile to reproject
+        base_srs = osr.SpatialReference()
+        # NAD83(CSRS) / UTM zone 10N
+        base_srs.ImportFromEPSG(3157)
+        extents = [
+            443723.1273278, 4956276.905980, 443993.1273278, 4956546.905980]
+
+        polygon_a = shapely.geometry.box(
+            extents[0], extents[1], extents[2], extents[3])

```

Comment:
FYI, python's argument unpacking to even further abbreviate this: `polygon_a = shapely.geometry.box(*extents)`.  What you have here will do exactly the same thing!

Reply:
You know I've seen you guys use this and thought about it, but must not have thought about it long enough to implement it. I will update it so that it helps me practice it.

Reply:
Done in [0f5e91](https://github.com/natcap/pygeoprocessing/pull/31/commits/0f5e91c7e1856f9ff8055b4bd8cff1516c412d99)